integration_test/third_party_apps_data/applications/mysql/debian_ubuntu/install

```diff
@@ -4,7 +4,7 @@ sudo apt update
 sudo apt install -y wget
 
 # From https://dev.mysql.com/doc/mysql-apt-repo-quick-guide/en/#repo-qg-apt-repo-manual-setup
-sudo apt-key adv --keyserver pgp.mit.edu --recv-keys 3A79BD29
+sudo apt-key adv --keyserver pool.sks-keyservers.net --recv-keys 3A79BD29
```

Comment:
Maybe use keyserver.ubuntu.com instead? See https://github.com/rvm/rvm/issues/5096.

Reply:
done. the latest snapshot tries both locations. sadface